schemas/com.snowplowanalytics.snowplow.storage/amazon_dynamodb_config/jsonschema/2-0-0

```diff
@@ -0,0 +1,52 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Snowplow Amazon DynamoDB storage configuration",
+    "self": {
+        "vendor": "com.snowplowanalytics.snowplow.storage",
+        "name": "amazon_dynamodb_config",
+        "format": "jsonschema",
+        "version": "2-0-0"
+    },
+    "type": "object",
+    "properties": {
+        "name": {
+            "type": "string"
+        },
+        "auth": {
+            "description": "Authentication method for Events Manifest",
+            "oneOf": [
+                {
+                    "type": "object",
+                    "properties": {
+                        "accessKeyId": {
+                            "type": "string"
+                        },
+                        "secretAccessKey": {
+                            "type": "string"
+                        }
+                    },
+                    "required": ["accessKeyId", "secretAccessKey"]
+                },
+                {
+                    "type": "null"
+                }
+            ]
+        },
+        "awsRegion": {
+            "type": "string"
+        },
+        "dynamodbTable": {
+            "type": "string"
+        },
+        "id": {
+            "type": "string",
+            "format": "uuid"
+        },
+        "purpose": {
+            "type": "string",
+            "enum": ["EVENT_MANIFEST"]
+        }
+    },
+    "additionalProperties": false,
+    "required": ["name", "auth", "awsRegion", "dynamodbTable", "purpose"]
```

Comment:
Let's make `id` required again. We made it optional in `1-0-1`, but it was a mistake.